data-prepper-plugins/otel-metrics-source/src/test/resources/testjson/test-standard-metrics.json

```diff
@@ -17,7 +17,6 @@
     "kind": "GAUGE",
     "flags": 0,
     "description": "testGaugeDescription",
-    "serviceName": null,
```

Comment:
THis should have stayed, but not using `null` which is somewhat invalid. All OTel resources SHOULD have a service name.